### ISSUE TITLE
`netlify dev` crashes: ReferenceError: hasRequiredFiles is not defined

### ISSUE THREAD

**Issue by wavebeem** [netlify/cli#315]
**- Do you want to request a *feature* or report a *bug*?**

Bug

**- What is the current behavior?**

`netlify dev` crashes with:

```
◈ Netlify Dev ◈
ReferenceError: hasRequiredFiles is not defined
    at Array.module.exports (C:/Users/brian/AppData/Roaming/nvm/v12.2.0/node_modules/netlify-cli/node_modules/netlify-dev-plugin/src/detectors/middleman.js:4:3)
    at module.exports.serverSettings (C:/Users/brian/AppData/Roaming/nvm/v12.2.0/node_modules/netlify-cli/node_modules/netlify-dev-plugin/src/detect-server.js:20:40)
    at DevCommand.run (C:/Users/brian/AppData/Roaming/nvm/v12.2.0/node_modules/netlify-cli/node_modules/netlify-dev-plugin/src/commands/dev/index.js:187:26)
    at DevCommand._run (C:/Users/brian/AppData/Roaming/nvm/v12.2.0/node_modules/netlify-cli/node_modules/@oclif/command/lib/command.js:35:31)
```

**- If the current behavior is a bug, please provide the steps to reproduce.**

- Use Windows 10
- Launch PowerShell
- git clone https://github.com/wavebeem/mockbrian.com
- npm i -g netlify-cli
- netlify login
- netlify dev

**- What is the expected behavior?**

It won't crash

**- Please mention your node.js, and operating system version.**

Windows 10 version 1809 build 17763.475
node v12.2.0
netlify-cli/2.11.17 win32-x64 node-v12.2.0

**Comment by edskeizer** [comment]
Experiencing the same issue on Mac OS with node v10.14.2

**Comment by fwestberg** [comment]
Same issue on macOS 10.14.4 with netlify-cli/2.11.17 darwin-x64 node-v11.10.1

**Comment by jgarber623** [comment]
This should be resolved upstream by netlify/netlify-dev-plugin#178 (and released as version 1.0.19).

I think that means this project needs a dependency bump PR…?

**Comment by sw-yx** [comment]
yes, i have just released v2.11.18, sorry for the temporarily broken release everyone.
